Failure on test_wbemcli.py on RHEL 7.3

I get this test error for test_wbemcli.py on my RHEL 7.3 system. It occurs on every testcase in the same way, and is probably all the same reason. It succeeds on my Ubuntu 14.04 system at home.

```
=================================== FAILURES ===================================
______________________ TestsContainer.test_def_namespace _______________________

self = <test_wbemcli.TestsContainer testMethod=test_def_namespace>

    def test(self):  # pylint: disable=missing-docstring
        """ The test method that is generated."""
        # create the path for the quit script. Required to
        # exit wbemcli after the test.
        quit_script_file = create_abs_path('wbemcli_quit_script.py')
    
        script_name = 'wbemcli.bat' if os.name == 'nt' else 'wbemcli'
    
        url = 'http://blah' if test_params.url is None \
            else test_params.url
    
        cmd = ('%s %s %s -s %s' % (script_name,
                                   url,
                                   test_params.cmd,
                                   quit_script_file))
    
        # Disable python warnings for wbemcli call
        # because some imports generate deprecated warnings
        # that appear in std_err when nothing expected
        bash_cmd = 'bash -c "set -o pipefail;  PYTHONPATH=. ' \
                   'PYTHONWARNINGS= %s"' % cmd
    
        proc = Popen(bash_cmd, shell=True, stdout=PIPE, stderr=PIPE)
        std_out, std_err = proc.communicate()
        exitcode = proc.returncode
    
        # print('bash_cmd: %s\nexitcode: %s std_out:\n%s\nstd_err:'
        #      '\n%s' % (bash_cmd, exitcode, std_out, std_err))
    
        if six.PY3:
            std_out = std_out.decode()
            std_err = std_err.decode()
    
        if test_params.expected_exitcode is not None:
            self.assertEqual(exitcode, test_params.expected_exitcode,
                             ('Test %s: Unexpected ExitCode Err. '
                              'Expected %s: cmd="%s": '
                              'exitcode %s: stderr=%s' %
                              (test_name, test_params.expected_exitcode,
                               cmd,
>                              exitcode, std_err)))
E           AssertionError: Test test_def_namespace: Unexpected ExitCode Err. Expected 0: cmd="wbemcli http://blah  -s /home/maiera/Projects/pywbem/repos/pywbem/testsuite/wbemcli_quit_script.py": exitcode 126: stderr=bash: /home/maiera/virtualenvs/pywbem27/bin/wbemcli: python: bad interpreter: No such file or directory

testsuite/test_wbemcli.py:167: AssertionError
```

--- a/makefile
+++ b/makefile
@@ -277,11 +277,9 @@ pylint: pylint.log
 	@echo '$@ done.'
 
 .PHONY: install
-install: _pip $(sdist_file)
-	mkdir tmp_install
-	tar -x -C tmp_install -f $(sdist_file)
-	sh -c "cd tmp_install/$(package_name)-$(package_version) && python setup.py install_os && python setup.py install"
-	rm -Rf tmp_install
+install: _pip
+	python setup.py install_os
+	pip install -e .
 	@echo 'Done: Installed pywbem into current Python environment.'
 	@echo '$@ done.'
 
@@ -377,7 +375,7 @@ endif
 
 $(test_log_file): makefile $(package_name)/*.py testsuite/*.py coveragerc
 	rm -f $(test_log_file)
-	bash -c "set -o pipefail; PYTHONWARNINGS=default PYTHONPATH=. py.test --cov $(package_name) $(coverage_report) --cov-config coveragerc --ignore=attic --ignore=releases --ignore=testsuite/testclient -s 2>&1 |tee $(test_tmp_file)"
+	bash -c "set -o pipefail; PYTHONWARNINGS=default py.test --cov $(package_name) $(coverage_report) --cov-config coveragerc --ignore=attic --ignore=releases --ignore=testsuite/testclient -s 2>&1 |tee $(test_tmp_file)"
 	mv -f $(test_tmp_file) $(test_log_file)
 	@echo 'Done: Created test log file: $@'
 

